qgisfeedproject/qgisfeed/utils.py

```diff
@@ -43,3 +44,53 @@ def get_field_max_length(ConfigurationModel: Model, field_name: str):
         return config.max_characters
     except ConfigurationModel.DoesNotExist:
         return 500
+
+
+def push_to_linkedin(title, content):
+    access_token = 'YOUR_ACCESS_TOKEN'
+    headers = {
+        'Authorization': f'Bearer {access_token}',
+        'Content-Type': 'application/json',
+        'X-Restli-Protocol-Version': '2.0.0'
+    }
+    payload = {
+        "author": "urn:li:person:YOUR_PERSON_URN",
+        "lifecycleState": "PUBLISHED",
+        "specificContent": {
+            "com.linkedin.ugc.ShareContent": {
+                "shareCommentary": {
+                    "text": title + content
+                },
+                "shareMediaCategory": "NONE"
+            }
+        },
+        "visibility": {
+            "com.linkedin.ugc.MemberNetworkVisibility": "PUBLIC"
+        }
+    }
+    response = requests.post('https://api.linkedin.com/v2/ugcPosts', headers=headers, json=payload)
+    return response.json()
+
+
+def push_to_facebook(title, content):
+    access_token = 'YOUR_ACCESS_TOKEN'
```

Comment:
@Xpirix how are you going to add the token?

Reply:
@dimasciput My bad, these functions are not used. I've removed them